math/genvector/doc/index.md

```diff
@@ -68,89 +71,83 @@ functionality.
-*   ROOT::Math::Polar3DVector vector based on r,theta,phi coordinates (polar) in double precision
-*   ROOT::Math::Polar3DVectorF vector based on r,theta,phi coordinates (polar) in float precision
-*   ROOT::Math::RhoZPhiVector vector based on rho, z,phi coordinates (cylindrical) in double precision
-*   ROOT::Math::RhoZPhiVectorF vector based on rho, z,phi coordinates (cylindrical) in float precision
-*   ROOT::Math::RhoEtaPhiVector vector based on rho,eta,phi coordinates (cylindrical using eta instead of z) in double precision
-*   ROOT::Math::RhoEtaPhiVectorF vector based on rho,eta,phi coordinates (cylindrical using eta instead of z) in float precision
+*   ROOT::Math::XYZVector vector based on x,y,z coordinates (cartesian)
+*   ROOT::Math::Polar3DVector vector based on r,theta,phi coordinates (polar)
+*   ROOT::Math::RhoZPhiVector vector based on rho, z,phi coordinates (cylindrical)
+*   ROOT::Math::RhoEtaPhiVector vector based on rho,eta,phi coordinates (cylindrical using eta instead of z)
+precision
 
 ### LorentzVector
 
 Type definitions for Lorentz vectors in four dimensions, based on ROOT::Math::LorentzVector, are defined by `Math/Vector4D.h`:
 
-*   ROOT::Math::XYZTVector vector based on x,y,z,t coordinates (cartesian) in double precision
-*   ROOT::Math::XYZTVectorF vector based on x,y,z,t coordinates (cartesian) in float precision
-*   ROOT::Math::PtEtaPhiEVector vector based on pt (rho),eta,phi and E (t) coordinates in double precision
-*   ROOT::Math::PtEtaPhiMVector vector based on pt (rho),eta,phi and M (t) coordinates in double precision
-*   ROOT::Math::PxPyPzMVector vector based on px,py,pz and M (mass) coordinates in double precision
-*   ROOT::Math::PxPyPzEVector vector based on px,py,pz and E (energy) coordinates in double precision
+*   ROOT::Math::XYZTVector vector based on x,y,z,t coordinates (cartesian)
+*   ROOT::Math::PtEtaPhiEVector vector based on pt (rho),eta,phi and E (t) coordinates
+*   ROOT::Math::PtEtaPhiMVector vector based on pt (rho),eta,phi and M (t) coordinates
+*   ROOT::Math::PxPyPzMVector vector based on px,py,pz and M (mass) coordinates
+*   ROOT::Math::PxPyPzEVector vector based on px,py,pz and E (energy) coordinates
 
-The metric used for all the LorentzVector's is (-,-,-,+)
+The metric used for any such LorentzVector is (-,-,-,+).
 
 
 \anchor GenVectorOperations
 ## Operations
 
 ### Constructors and Assignment
 
-Vectors can be constructed by passing its coordinate representation.
-In addition the vector classes can be constructed by any vector which implements the
-accessors x(), y() and z(). This can be another vector based on a different coordinate
-system type or even any vector of a different package, like the CLHEP `Hep3Vector` that
-implements the required signatures.
+A vector can be constructed from its coordinate representation:
+
+~~~{.cpp}
+ROOT::Math::PtEtaPhiMVector v1(10. /*pt*/, 0.1 /*eta*/, 0.24 /*phi*/, 5 /*M*/);
+~~~
```

Comment:
If we put an example upfront (we decided against), you can drop the "plain" constructors in the second example

Reply:
I'm not sure I understand what you mean here, can you expand? Are you proposing to remove `XYZVector v1(1,2,3);`?

I thought I'd suggest to put this first because "constructed from its coordinate representation" is fairly cryptic, and an example makes it immediately obvious (if the example follows immediately).

Reply:
Yes, that's what I was thinking about, now we have "plain" constructors in the second example. But they are for other coordinate systems and show the conversion from XYZ to RhoEtaPhi, so we might as well leave them.